lib/rules/value-keyword-case/index.js

```diff
@@ -4,6 +4,7 @@ const keywordSets = require("../../reference/keywordSets")
 const declarationValueIndex = require("../../utils/declarationValueIndex")
 const getUnitFromValueNode = require("../../utils/getUnitFromValueNode")
 const isCounterIncrementCustomIdentValue = require("../../utils/isCounterIncrementCustomIdentValue")
+const isCounterResetCustomIdentValue = require("../../utils/isCounterIncrementCustomIdentValue")
```

Comment:
If you're going to separate increment and reset you should use separate utils that use the respective keywordSets.

But I think it would better to just merge them into one set: keywordSets.counterKeywords / isCounterCustomIdentValue as they are identical in the spec: https://www.w3.org/TR/CSS2/generate.html#propdef-counter-reset (and also in CSS3 where counter-set is proposed)